### DISCUSSION TITLE
Add up the amounts of the same ingredients

### DISCUSSION THREAD

**Discussion by Phily757** [TandoorRecipes/recipes#2200] [Q&A] [answered]
Hello Tandoor community!

I was testing Tandoor Recipes and so far I'm very impressed. But one thing is bothering me.
If I'm adding the **same ingredient** to **several steps** (e.g. water; 100 ml in the first step and later another 50 ml) and i enable the option 'Display a list of all ingredients at the start of the recipe' then not the total amount (150 ml) is displayed. Instead 'water' is listed twice (100 ml and 50 ml).
Not sure if this is a bug or intended behaviour? For me it would be nice to add an option (besides 'Display a list of all ingredients at the start of the recipe.') to enable adding up the amounts of the same ingredients.

**Comment by vabene1111** [comment, marked as answer]
this is a feature not a bug, there is a FR somewhere that plans to change this

**Comment by LimesRunner** [comment]
I just wanted to write the same topic as idea. It would really be very helpful.

**Comment by patrickwachowiak** [comment]
I'd find adding up ingredients of the same type in the topmost list also more reasonable. It would also be more consistent with adding recipes to the shopping list...there, ingredients get added up already.
